src/components/BubbleWrapper.tsx

```diff
@@ -191,7 +192,6 @@ const BubbleWrapper = (props: PropsWithChildren<ChatbotBubbleWrapperPropsType> &
             </Animated.View>
           </Animated.View>
         </Animated.View>
-        <Animated.View style={[animatedSpacerStyles]} />
```

Comment:
@CharlesMangwa actually I'm not sure why it works before my changes, because it's rendered at the bottom of the tree despite the fact that the first element in the group needs a top!!! margin of 24, see this chunk of code:
```
 const animatedSpacerStyles = useAnimatedStyle(() => {
    const height = isFirstInGroup ? 24 : 2
    return {
      borderWidth: 1,
      height: withSpring(animateBubble.value || animateBubbleIntro.value ? height : 0, DEFAULT_SPRING_CONFIG),
    }
  })
```

Another way of accomplishing correct behavior would be keeping the spacers where they are, but flipping the height ternary to ```const height = isFirstInGroup ? 2 : 24```

Reply:
oh okok, gotcha! thanks for the explanation! 🙌